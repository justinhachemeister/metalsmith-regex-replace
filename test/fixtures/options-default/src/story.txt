There once was a dog named Spot. Spotting a cat named |Spot|, he barked. The cat
ran up a tree. Spot stopped in a shady |spot| under the tree and barked into the
air.

Spot's owner heard the commotion and walked to the spot where he heard the all
the ruckus. "SPOT! COME HERE!", yelled the owner.