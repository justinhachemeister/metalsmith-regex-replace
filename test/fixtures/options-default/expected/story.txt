There once was a dog named Rex. Spotting a cat named Spot, he barked. The cat
ran up a tree. Rex stopped in a shady spot under the tree and barked into the
air.

Rex's owner heard the commotion and walked to the rex where he heard the all
the ruckus. "REX! COME HERE!", yelled the owner.